joomla/joomla-cms
# [4.0][mod_messages] messages badge

Pull Request for Issue #26895 .

### Summary of Changes
fix


### Testing Instructions

1.   apply pr
2.     Login to Joomla admin with one user
3.     Go to the private message section.
4.     Add another user Id in recipients
5.     Add some context and send the message.
6.     Now login with another user.
7.     See the admin panel



### Expected result
badge


### Actual result

no badge
